odoo/tools/rendering_tools.py

```diff
@@ -42,33 +42,34 @@ def parse_inline_template(text):
     for match in INLINE_TEMPLATE_REGEX.finditer(text):
         literal = text[current_literal_index:match.start()]
         expression = match.group(1)
-        groups.append((literal, expression))
+        default = match.group(3)
+        groups.append((literal, expression.strip(), default or ''))
         current_literal_index = match.end()
 
     # string past last regex match
     literal = text[current_literal_index:]
     if literal:
-        groups.append((literal, ''))
+        groups.append((literal, '', ''))
 
     return groups
 
 def convert_inline_template_to_qweb(template):
     template_instructions = parse_inline_template(template or '')
     preview_markup = []
-    for string, expression in template_instructions:
+    for string, expression, default in template_instructions:
         if expression:
-            preview_markup.append(Markup('{}<t t-out="{}"/>').format(string, expression))
+            preview_markup.append(Markup('{}<t t-out="{}">{}</t>').format(string, expression, default))
```

Comment:
Question:
Do we absolutely need to use the format on Markup, or could we use f-strings here?